sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -639,7 +639,7 @@ case class DescribeTableCommand(
  *    select * from (from a select * select *)
  * 7. Common table expressions (CTEs)
  */
-case class DescribeQueryCommand(query: LogicalPlan)
+case class DescribeQueryCommand(queryText: String)(query: LogicalPlan)
```

Comment:
Can't we just have:

```scala
case class DescribeQueryCommand(queryText: String, query: LogicalPlan)
```

Reply:
Ah, I see. The logical plan will be hidden.

Reply:
This is hacky, can we override toString instead?

Reply:
+1

Reply:
@cloud-fan @viirya Oh.. i think i did try it. Let me see why it didn't work.

Reply:
@cloud-fan @viirya Thanks for the suggestion. I did override `simpleString` to get it to work. Please take a look when you get a chance.